### PR TITLE
Various Tweaks

### PR DESCRIPTION
**Pull request by jonesmz**
Fixes https://github.com/GenPi64/Build.Dist/issues/120 and https://github.com/GenPi64/Build.Dist/issues/130 , pretty sure.

### CODE REVIEW

File: chroot.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #! /bin/bash
 
+export SYSTEMD_NSPAWN_LOCK=0
```
Comment (jonesmz) [inline chroot.sh:3]: Useful for allowing a human to chroot into the image while the creation process is still on-going.

File: config/pyconfig/GenPi64Config.py [hunk 2]
```diff
@@ -39,8 +39,12 @@
     ],
     "portage": Base["portage"] | {
         "make.conf": Base["portage"]["make.conf"] | {
-            "CHOST":"aarch64-unknown-linux-gnu",
-            "MAKEOPTS":"-j4 -l4",
+            "CFLAGS": "${CFLAGS} -march=armv8-a+crc -mtune=cortex-a72 -ftree-vectorize -O2 -pipe",
```
Comment (jonesmz) [inline config/pyconfig/GenPi64Config.py:42]: consistency with the profile from the genpi64-overlay.

File: scripts/chroot.py [hunk 3]
```diff
@@ -27,6 +27,8 @@
     elif chroot_cmd == 'systemd-nspawn':
         os.execvpe('systemd-nspawn',
                    ['systemd-nspawn',
+                    f'--timezone=off',
```
Comment (jonesmz) [inline scripts/chroot.py:30]: Prevent systemd-nspawn from "helping" for the value of /etc/localtime

File: scripts/chroot.py [hunk 4]
```diff
@@ -27,6 +27,8 @@
...
+                    f'--resolv-conf=bind-host',
```
Comment (jonesmz) [inline scripts/chroot.py:31]: Prevent systemd-nspawn from "helping" for the value of /etc/resolv.conf

**Review by samip5 — APPROVED** [review]

**Pull request merged**